From 8706dbd284ba1e31a661b96f9cbc94c9daccfb39 Mon Sep 17 00:00:00 2001
From: muzena <trebelnik2@gmail.com>
Date: Thu, 12 Mar 2020 21:46:48 +0100
Subject: [PATCH] Restore requirements

---
 setup.py | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/setup.py b/setup.py
index 3d9f574..6a967cc 100755
--- a/setup.py
+++ b/setup.py
@@ -73,13 +73,7 @@ battery status.
 			],
 
 		platforms=['linux'],
-
-		# sudo apt install python-gi python3-gi \
-		#        gir1.2-gtk-3.0 gir1.2-notify-0.7 gir1.2-appindicator3-0.1
-		# os_requires=['gi.repository.GObject (>= 2.0)', 'gi.repository.Gtk (>= 3.0)'],
-
-		python_requires='>=3.2',
-		install_requires=['pyudev (>= 0.13)', ],
+		requires=['pyudev (>= 0.13)', 'gi.repository.GObject (>= 2.0)', 'gi.repository.Gtk (>= 3.0)'],
 		package_dir={'': 'lib'},
 		packages=['hidapi', 'logitech_receiver', 'solaar', 'solaar.ui', 'solaar.cli'],
 		data_files=list(_data_files()),
-- 
2.17.1

