Description: Restore Python requirements for distutils
Author: Stephen Kitt <skitt@debian.org>

--- a/setup.py
+++ b/setup.py
@@ -71,12 +71,7 @@
 			],
 
 		platforms=['linux'],
-
-		# sudo apt install python-gi python3-gi \
-		#        gir1.2-gtk-3.0 gir1.2-notify-0.7 gir1.2-appindicator3-0.1
-		# os_requires=['gi.repository.GObject (>= 2.0)', 'gi.repository.Gtk (>= 3.0)'],
-
-		install_requires=['pyudev (>= 0.13)', ],
+		requires=['pyudev (>= 0.13)', 'gi.repository.GObject (>= 2.0)', 'gi.repository.Gtk (>= 3.0)'],
 		package_dir={'': 'lib'},
 		packages=['hidapi', 'logitech_receiver', 'solaar', 'solaar.ui', 'solaar.cli'],
 		data_files=list(_data_files()),
